src/Nethermind/Nethermind.Blockchain.Test/Validators/HeaderValidatorTests.cs

```diff
@@ -43,25 +44,26 @@ public class HeaderValidatorTests
         private TestLogger _testLogger;
         private Block _parentBlock;
         private Block _block;
+        private IBlockTree _blockTree;
 
         [SetUp]
         public void Setup()
         {
             DifficultyCalculator calculator = new DifficultyCalculator(new SingleReleaseSpecProvider(Frontier.Instance, ChainId.Mainnet));
             _ethash = new EthashSealValidator(LimboLogs.Instance, calculator, new CryptoRandom(), new Ethash(LimboLogs.Instance));
             _testLogger = new TestLogger();
-            var blockInfoDb = new MemDb();
-            BlockTree blockStore = new BlockTree(new MemDb(), new MemDb(), blockInfoDb, new ChainLevelInfoRepository(blockInfoDb), FrontierSpecProvider.Instance, Substitute.For<IBloomStorage>(), LimboLogs.Instance);
+            MemDb blockInfoDb = new MemDb();
+            _blockTree = new BlockTree(new MemDb(), new MemDb(), blockInfoDb, new ChainLevelInfoRepository(blockInfoDb), FrontierSpecProvider.Instance, Substitute.For<IBloomStorage>(), LimboLogs.Instance);
```

Comment:
it would be better to go an opposite direction to make tests parallelizable (see the TestCtx examples from other test classes)

Reply:
Agree. I built it on existing tests. Unfortunately, I think we have many such tests, but it should be quite easy to refactoring.